eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -773,7 +773,7 @@ ezpublish_rest_loadRoleDraft:
 ezpublish_rest_loadRoleDraftByRoleId:
     path: /user/roles/{roleId}/draftByRoleId
```

Comment:
So unless I'm mistaken, given that there is a role with id #1, and a role draft for role #1 (role id) with id #2 (role draft id):

```
/user/roles/{roleId}/draft
/user/roles/{roleDraftId}/draft
/user/roles/{roleDraftId}
```

Would all return the same object ? Not so sure about the last one, though.

i'm really confused here.